lib/offline/LogSessionDataProvider.cpp

```diff
@@ -140,7 +140,7 @@ namespace MAT_NS_BEGIN
         }
         std::vector<std::string> v;
         StringUtils::SplitString(content, '\n', v);
-        if (v.size() != 2) {
+        if (v.size() != 3) {
```

Comment:
Is there a compat concern (e.g. the file was written by a previous version of the sdk, and the reader is new version)?

Reply:
No, it should be fine as we are only changing the parsing logic in the new sdk. The file was correctly written with older sdk, and newer sdk ensure that it is now read correctly too.

Reply:
Thanks.